std/typecons.d

```diff
@@ -1328,6 +1349,48 @@ if (distinctFieldNames!(Specs))
                         Unqual!(typeof(this)).stringof ~ "', not '%" ~ fmt.spec ~ "'.");
             }
         }
+
+        ///
+        static if (Types.length == 0)
+        @safe unittest
+        {
+            import std.format : format, FormatException;
+            import std.exception : assertThrown;
+
+            // enum tupStr = tuple(1, 1.0).toString; // toString is *impure* for double (issue 17628)
+            // static assert(tupStr == `Tuple!(int, double)(1, 1)`);
+
+            Tuple!(int, double)[3] tupList = [ tuple(1, 1.0), tuple(2, 4.0), tuple(3, 9.0) ];
```

Comment:
Wasn't this already part of #6502?

Reply:
It should have been. Now, it's in #6504.